pkg/catalog/interrogate.go

```diff
@@ -196,6 +197,13 @@ func ImageForRole(
 	)
 }
 
+// ImageForStatefulSet returns the image is currently used at a given statefulSet
```

Comment:
Probably makes more sense to have this in pkg/shared/util.go ... not really a catalog-related thing.

Reply:
Well, I removed it and added to `pkg/shared/util.go` more common function `StatefulSetContainers` returns the containers array of a statefulset spec